doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -101,6 +106,11 @@ To edit clusters using the ZAP tool, complete the following steps:
 
       west zap-gui
 
+
+   .. note::
+      The ZAP tool UI may vary depending on the ZAP version. The following steps should be considered as guidelines.
```

Comment:
I think there is one line - one sentence rule for the .rst files, so the second sentence should be started in the new line.